addons/api/package.json

```diff
@@ -31,8 +31,8 @@
     "ember-cli-babel": "^7.26.10",
     "ember-cli-htmlbars": "^6.1.0",
     "ember-cli-mirage": "^2.2.0",
+    "ember-copy": "^2.0.1",
```

Comment:
Asking without the context of using `ember-copy` within this PR: why we moved from devDeps to dep?

Reply:
see this thread: https://hashicorp.slack.com/archives/C04BFSLA2L9/p1669830260871929

Reply:
Aha! that makes sense 👍